issue-triaging.md

```diff
@@ -71,3 +71,52 @@ triaged and is ready for work. If someone who is assigned to an issue is no long
 | `Sponsor Needed`     | A new component has been proposed, but implementation is not ready to begin. This can be because a sponsor has not yet been decided, or because some details on the component still need to be decided.        |
 | `Accepted Component` | A sponsor has elected to take on a component and implementation is ready to begin.                                                                                                                             |
 | `Vendor Specific Component` | This should be applied to any component proposal where the functionality for the component is particular to a vendor.                                                                                          |
+
+### State diagram
+
+Here is a diagram outlining the potential issue states and how issues move through different stages:
+
+```mermaid
+flowchart TD
+ subgraph graph1["**needs-triage**"]
+        n1(["Needs Triage"])
+  end
+ subgraph graph2["**waiting-for-codeowners**"]
+        n3["Waiting for Codeowners<br>to further validate the issue"]
+  end
+ subgraph graph3["**waiting-for-author**"]
+        n4["Waiting for author to provide more details"]
+  end
+ subgraph graph4["**help-wanted**"]
+        n8["Mark as help-wanted"]
+  end
+    n1 --> n2["Has good repro steps <br>and/or description?"]
+    n2 -- Yes --> n3
+    n2 -- No/Need more details --> n4
+    n2 -- Invalid configuration/alternative available --> n5(["Provide a detailed comment on the issue and close it."])
+    n3 --> n6["Codeowner has time<br>to fix it?"]
```

Comment:
maybe add an arrow for codeowners to reject the issue, in case that's not clear.

Reply:
I see, thanks for this. I'll add that

Reply:
@atoulme once you're back, pleas take a look!